.github/workflows/functional.yml

```diff
@@ -27,14 +27,9 @@ jobs:
         ports:
           - 27017:27017
 
-      mosquitto:
-        # Needed by MQTT notification tests
-        image: eclipse-mosquitto:1.6.15
-        ports:
-        - 1883:1883
-
       mosquitto-extra:
         # Needed by MQTT notification tests (multi broker)
+        #image: eclipse-mosquitto:2.0.11
         image: eclipse-mosquitto:1.6.15
```

Comment:
The MQTT broker in 1883 uses 2.0.11 version without problem (see L59), but this auxiliar MQTT broker in 1884 has to be 1.6.15 or 3001_mqtt/mqtt_subscription_multibroker.test will break.

It's weird... I'm looking at it.